### PR TITLE
AppDaemon recipe

### PR DESCRIPTION
**Pull request by persuader72**
Hi , I added to the repository this nice addon (http://appdaemon.readthedocs.io/en) for hass. I like the dashboard very much ;)
Do you think that can be useful to integrate it into the main repository?

Regards 
Stefano

### CODE REVIEW

**Comment by bachp** [comment]
I didn't know about appdaemon but I will check it out. I'm currently at ELCE so I need to see when I find some time.

**Comment by persuader72** [comment]
Oh It sounds very interesting!

Have a great time
Stefano

**Comment by bachp** [comment]
@persuader72 If I understand correctly one can install appdaemon on the target in addition to home-assistant and then use its features. If yes I think it makes sense to put this into the repo.

Can you also submit this against master. I want to avoid having features in the pyro branch that are not in master.

**Comment by persuader72** [comment]
@bachp Yes, the app is running as an independent daemon and communicates with HA only through its api, moreover it is listed in the root of the HA github account so it can be truly considered within the HA ecosystem itself.

I will submit a PR against master, but I can't test it because I do not have a real environment that run on master neither a simulated one. If it's possible you'll have to check that the new recipes will compile correctly in your environment.

File: recipes-devtools/python/python-aiohttp-jinja2.inc [hunk 1]
```diff
@@ -0,0 +1,15 @@
+SUMMARY = "jinja2 template renderer for aiohttp.web (http server for asyncio)"
+HOMEPAGE = "https://github.com/aio-libs/aiohttp_jinja2/"
+LICENSE = "Apache 2"
```
Comment (bachp) [inline recipes-devtools/python/python-aiohttp-jinja2.inc]: Should be `Apache-2.0`

File: recipes-devtools/python/python-sseclient.inc [hunk 2]
```diff
@@ -0,0 +1,11 @@
+SUMMARY = "Python client library for reading Server Sent Event streams."
+HOMEPAGE = "https://github.com/btubbs/sseclient"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://README.rst;md5=17055b849ff906df0ec96960ab2476df"
```
Comment (bachp) [inline recipes-devtools/python/python-sseclient.inc]: This is not the correct license file.

If this happens I usually submit a merge request fixing the issue upstream and add a `#TODO`.

For this I already opened one at https://github.com/btubbs/sseclient/pull/11
Could you please add `#TODO: https://github.com/btubbs/sseclient/pull/11`

**Comment by persuader72** [comment]
If you agree, I would want remove this pull request and repeat in a few days because i want to add the the service init files for appdaemon. I will repeat it against master branch.

**Comment by persuader72** [comment]
Leveled branch pyro to master branch

**Pull request merged**